charts/prometheus-stackdriver-exporter/templates/deployment.yaml

```diff
@@ -57,6 +57,13 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          
```

Comment:
```suggestion

```

Reply:
Thank you for the review. I have bumped the chart patch version, and added DCO to my last two commits. Do I need to rewrite history or create a new branch for my first two commits that I made in the GitLab UI that don't have DCO?